docs/reference/commandline/plugin_set.md

```diff
@@ -27,20 +27,67 @@ Options:
 
 Change settings for a plugin. The plugin must be disabled.
 
+The settings currently supported are:
+ * env variables
+ * source of mounts
+ * path of devices
+ * args
 
-The following example installs change the env variable `DEBUG` of the
+The following example change the env variable `DEBUG` on the
 `no-remove` plugin.
 
 ```bash
-$ docker plugin inspect -f {{.Config.Env}} tiborvass/no-remove
+$ docker plugin inspect -f {{.Settings.Env}} tiborvass/no-remove
 [DEBUG=0]
 
 $ docker plugin set DEBUG=1 tiborvass/no-remove
 
-$ docker plugin inspect -f {{.Config.Env}} tiborvass/no-remove
+$ docker plugin inspect -f {{.Settings.Env}} tiborvass/no-remove
 [DEBUG=1]
 ```
 
+The following example change the source of the `mymount` mount on
+the `myplugin` plugin.
+
+```bash
+$ docker plugin inspect -f '{{with $mount := index .Settings.Mounts 0}}{{$mount.Source}}{{end}}' myplugin
+/foo
+
+$ docker plugins set mymount.source=/bar myplugin
```

Comment:
Pluginname should preceed the setting. Should be `docker plugins set myplugin mymount.source=/bar `